src/STEMMUS_SCOPE.m

```diff
@@ -815,7 +803,11 @@
             hSAVE = hN;
         end
     end
-    run Forcing_PARM;
+
```

Comment:
Perhaps a short comment here explaining why the variables are (re)set here could be useful.

Reply:
Done!